accelerator-core/build.gradle

```diff
@@ -66,9 +66,9 @@ dependencies {
     compile 'com.android.support:appcompat-v7:27.0.2'
     compile 'com.opentok.android:opentok-solutions-logging:+'
     if (isPreview()) {
-        compile 'com.opentok.android:opentok-android-sdk-preview:2.13.+'
+        compile 'com.opentok.android:opentok-android-sdk-preview:2.14.+'
```

Comment:
You may have a variable with 2.13 and the preview will be that variable + 1. That way you just need to update one value

Reply:
+1